lib/block-patterns.php

```diff
@@ -158,3 +158,62 @@
 							<!-- /wp:social-links -->',
 	)
 );
+
+add_action(
+	'init',
+	function() {
+
+		$core_block_patterns = array(
```

Comment:
It might be helpful to add a comment about the plan to replace these in core. Reading this code before reading the entire thread of this PR, I was confused why we were defining two arrays of block patterns and unregistering the first set.

Reply:
Added here: https://github.com/WordPress/gutenberg/pull/29973/commits/18480b3fe0c6d0dc738faad2499399826919f79b

We'll also make a note in the Core ticket once this has been merged.